### PR TITLE
Fix some issues on the documentation

### PR DESCRIPTION
**Pull request by palexster**
# Description

This PR adds some refactors to the installation section of the documentation

### CODE REVIEW

File: docs/pages/User/Install/platforms/k3s.md [hunk 1]
```diff
@@ -63,35 +50,37 @@ The most important values you can set are the following:
...
-```
+#### On-Premise to On-Premise with direct connectivity
 
-or ALTERNATIVELY pass the desired parameters as extra-arguments:
+If you want to connect your cluster with another K3s/K8s in the same LAN, you do not need further configuration. You can install Liqo with just specifying the correct values for the three variables mentioned above:
 
-```bash
-helm install liqo liqo-helm/liqo --set clusterName="MyCluster" --set networkManager.config.podCIDR="10.42.0.0/16" --set networkManager.config.serviceCIDR="10.43.0.0/16" ...
 ```
+helm install liqo liqo-helm/liqo --set clusterName="MyCluster" --set networkManager.config.podCIDR="10.42.0.0/16" --set networkManager.config.serviceCIDR="10.43.0.0/16"
```
Comment (giorio94) [inline docs/pages/User/Install/platforms/k3s.md:58]: `liqo-helm` -> `liqo`

File: docs/pages/User/Install/platforms/aks.md [hunk 2]
```diff
@@ -120,20 +105,19 @@ You can install Liqo using helm 3.
...
 If you are installing Liqo for the first time, you can download the default values.yaml file from the chart.
 
 ```bash
-helm fetch liqo-helm/liqo --untar
-less ./liqo/values.yaml
+helm show values liqo/liqo > ./values.yaml
 ```
 
-After, modify the ```./liqo/values.yaml``` as specified above to obtain the desired configuration and install Liqo.
+After modifying the values.yaml with the desired values, as described in [previous section](#setup-values), you can perform the Liqo installation by typing:
```
Comment (giorio94) [inline docs/pages/User/Install/platforms/aks.md:117]:
```suggestion
After modifying the `values.yaml` file with the desired values, as described in [the previous section](#setup-values), you can perform the Liqo installation by typing:
```

**Review by giorio94 — COMMENTED** [review]
Some small comments here and there

**Review by alacuku — APPROVED** [review]

**Review by mlavacca — APPROVED** [review]

**Pull request merged**